nexus/src/external_api/views.rs

```diff
@@ -147,6 +147,46 @@ impl Into<VpcSubnet> for model::VpcSubnet {
     }
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, Serialize, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum VpcRouterKind {
+    System,
+    Custom,
+}
+
+impl Into<VpcRouterKind> for model::VpcRouterKind {
```

Comment:
Is it possible to `impl From<model::VpcRouterKind> for VpcRouterKind`?

I think it's functionally identical to this, but it's [preferable to impl From over Into when given the option](https://doc.rust-lang.org/std/convert/trait.From.html):

> One should always prefer implementing From over [Into](https://doc.rust-lang.org/std/convert/trait.Into.html) because implementing From automatically provides one with an implementation of [Into](https://doc.rust-lang.org/std/convert/trait.Into.html) thanks to the blanket implementation in the standard library.
> Only implement [Into](https://doc.rust-lang.org/std/convert/trait.Into.html) when targeting a version prior to Rust 1.41 and converting to a type outside the current crate. From was not able to do these types of conversions in earlier versions because of Rust’s orphaning rules.

Reply:
Yep, definitely. We should change them all.

Reply:
Done in [636c72d](https://github.com/oxidecomputer/omicron/pull/776/commits/636c72d7367ef739dcb4d5eff29dc66d58046ceb). I'll change the others in a separate PR to reduce noise.